tests/components/plex/test_init.py

```diff
@@ -74,63 +80,96 @@
     )
 
 
-async def test_setup_with_config_entry(hass, caplog):
-    """Test setup component with config."""
+class TestClockedPlex(ClockedTestCase):
```

Comment:
Too bad that pytest hasn't completed a fixture with time advancing functionality. I think it's ok to use this test class for this purpose only until we have an alternative.

Reply:
I'm open to other alternatives, but this is the best I was able to find.

Reply:
Yeah, I haven't found anything either. Just a stale PR in pytest-asyncio. Let's go with this for now.